maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/NullConsoleReporter.java

```diff
@@ -33,7 +33,10 @@
 class NullConsoleReporter
     extends ConsoleReporter
 {
-    NullConsoleReporter() {
+
+    static final NullConsoleReporter INSTANCE = new NullConsoleReporter();
+
+    private NullConsoleReporter() {
```

Comment:
here as well
pls run the build. it explores checkstyle erros.
Did you get your IntelliJ Idea now working with our Maven checkstyle?